consensus/polybft/signer/signature_test.go

```diff
@@ -26,6 +27,54 @@ func Test_VerifySignature(t *testing.T) {
 	assert.False(t, signature.Verify(blsKey.PublicKey(), invalidTestMsg))
 }
 
+func Test_VerifySignature_NegativeCases(t *testing.T) {
+	t.Parallel()
+
+	validTestMsg := testGenRandomBytes(t, 32)
+
+	blsKey, _ := GenerateBlsKey()
```

Comment:
Let's check an error, doesn't harm I guess

```suggestion
	blsKey, err := GenerateBlsKey()
    require.NoError(t, err)
```

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1214/commits/19a511499342a773c277798166b2cc5feef543ec